integration/utils/contLoadGenerator.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/onflow/flow-go/module/metrics"
 
 	flowsdk "github.com/onflow/flow-go-sdk"
-	"github.com/onflow/flow-go-sdk/client"
+ "github.com/onflow/flow-go-sdk/access"
```

Comment:
run gofmt ...

Reply:
interesting... I have it automatic on save, looks like an IDE failure, should've noticed it.